### PR TITLE
Changelog for v0.8.0

### PR DESCRIPTION
**Pull request by laurenzlong**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+important - `firebase-admin` peer dependency is now v5.5.0. Must update `firebase-admin` version inside of your function's directory's `package.json` and re-install locally.
```
Comment (justinrosenthal) [inline changelog.txt]: function's --> functions?

File: changelog.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+important - `firebase-admin` peer dependency is now v5.5.0. Must update `firebase-admin` version inside of your function's directory's `package.json` and re-install locally.
+changed - For Firestore events where no previous data exists, event.data.previous is no longer null, it is always a Firebase DocumentSnapshot. event.data.previous.exists will be false if there is no previous data. (This change was previously released in v4.7.4, and reverted in v4.7.5.)
```
Comment (justinrosenthal) [inline changelog.txt]: v4.7.4/5 --> v0.7.4/5

File: changelog.txt (hunk 1 above)
Comment (inlined) [reply changelog.txt]: +1

File: changelog.txt (hunk 2 above)
Comment (laurenzlong) [reply changelog.txt]: good catch!

**Pull request merged**